src/components/data-table/data-table.stories.js

```diff
@@ -0,0 +1,374 @@
+import { html } from "lit-html";
+import "../data-table";
+import { stylesheet, docsIntro } from "../../../.storybook/utils.js";
+import { args, argTypes } from "./config";
+import styles from "./styles.scss";
+
+export default {
+  title: "native-elements/vl-data-table",
+  decorators: [(story) => html`${stylesheet(styles)}${story()}`],
+  args,
+  argTypes,
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
+          stylesheets: ["data-table"],
+          root: "data-table",
+          intro:
+            "Use a data table to display (large quantities of) related data in a structured way.",
+        }),
+      },
+    },
+  },
+};
+
+export const Default = ({
```

Comment:
De data-vl-lined doet bij mij niets

Reply:
Het is de bedoeling dat je in 1 story de attributen kan togglen van een component en zo zijn variaties kan zien.
Nu is dat deels opgevangen in de 'default', maar dan begrijp ik niet dat er nog stories bijstaan met dezelfde html opmaak, maar gewoon met bepaalde attributen hardcoded. De joined row story lijkt me dan wel logisch want daar is er meer veranderd dan enkel de attributen.

Reply:
lined is blijkbaar veranderd bij DV naar "grid" dus ik heb de code aangepast.
De stories zijn ook verwijderd die geen verschillende markup hadden.